dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -369,11 +371,11 @@ public void activateDeactivateIndex () throws Exception {
 
         //Build the index names
         String timeStamp = String.valueOf( new Date().getTime() );
-        String workingIndex = ContentletIndexAPIImpl.ES_WORKING_INDEX_NAME + "_" + timeStamp;
-        String liveIndex = ContentletIndexAPIImpl.ES_LIVE_INDEX_NAME + "_" + timeStamp;
+        String workingIndex = IndexType.WORKING.getPrefix() + "_" + timeStamp;
+        String liveIndex = IndexType.LIVE.getPrefix() + "_" + timeStamp;
 
-        String oldActiveLive = indexAPI.getActiveIndexName(ContentletIndexAPI.ES_LIVE_INDEX_NAME);
-        String oldActiveWorking = indexAPI.getActiveIndexName(ContentletIndexAPI.ES_WORKING_INDEX_NAME);
+        String oldActiveLive = indexAPI.getActiveIndexName(IndexType.LIVE.getPrefix());
+        String oldActiveWorking = indexAPI.getActiveIndexName(IndexType.WORKING.getPrefix());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'oldActiveWorking' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)